source/includes/_webhooks.md

```diff
@@ -581,17 +581,13 @@ This event is fired two times in the lifecycle of a dividend post request.
         "currency": "GBP",
```

Comment:
further update on line 566-ish, you might want to update this description here - it will be outdated given your new webhook changes that will be going in.